tools/run_wasm

```diff
@@ -63,4 +63,4 @@ while [ $# -gt 0 ]; do
   esac
 done
 
-exec "$SHELL_BIN" "$RUN_WASM" $SHELL_ARG_SEPERATOR "$MJS_FILE" "$WASM_FILE" -- "$@"
+exec "$SHELL_BIN" $SHELL_OPTIONS "$RUN_WASM" $SHELL_ARG_SEPERATOR "$MJS_FILE" "$WASM_FILE" -- "$@"
```

Comment:
For arrays I believe this has to be `"${SHELL_OPTIONS[@]}"`.

Reply:
Thanks, done.